content/agent/basic_agent_usage/osx.md

```diff
@@ -81,50 +90,53 @@ You can either download the DMG package and install it manually, or use the one-
 2. Install the DMG package
 3. Add your API key to `/opt/datadog-agent/etc/datadog.yaml`
 
-Then start the Datadog Agent app (once started, you should see it in the system tray), and manage the Agent from there. The Agent6 also ships a web-based GUI to edit the Agent configuration files and much more, refer to the [changes and deprecations document][changes] document for more information.
+Then start the Datadog Agent app (once started, you should see it in the system tray), and manage the Agent from there. Agent v6 includes a web-based GUI to edit the Agent configuration files and much more.
+
 ### Install script
-#### To Upgrade
 
-The Agent 6.x installer can automatically convert your 5.x style Agent configuration at upgrade:  
+#### To Upgrade
 
+The Agent v6 installer can automatically convert v5 configurations during the upgrade:
 ```shell
-  DD_UPGRADE=true bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_mac_os.sh)"
```

Comment:
https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_mac_os.sh is correct for macOS

Reply:
fixed 👇